zh-CN/mapping_dev.zh_CN

```diff
@@ -48,95 +48,95 @@ dlgProgress.tle"JIRA"=JIRA
 dlgProgress.tle"Jump To"=跳转到
 dlgProgress.tle"Refresh"=刷新
 dlgProgress.tle"Reveal Commit"=显示提交
-dlgProgress.tle"SmartGit Installation Update"=SmartGit安装更新
-dlgQBugFileSendingFailed.fur%1=也许您需要配置代理才能访问我们服务器或我们的服务器暂时关闭.\n详情: $1
-dlgQBugFileSendingFailed.hdl%1=无法将崩溃日志发送到$1
+dlgProgress.tle"SmartGit Installation Update"=SmartGit 安装更新
+dlgQBugFileSendingFailed.fur%1=也许您需要配置代理才能访问我们服务器或我们的服务器暂时关闭。\n详情：$1
+dlgQBugFileSendingFailed.hdl%1=无法将崩溃日志发送到 $1
 dlgQBugFileSendingFailed.tle=本机崩溃日志
 dlgQBugReportSend.btn"Force Exit"=强制退出
-dlgQBugReportSend.btn"Report Bug"=反馈Bug
+dlgQBugReportSend.btn"Report Bug"=反馈 Bug
 dlgQBugReportSend.edt"Comments or Steps to Reproduce"=重现注释或步骤
 dlgQBugReportSend.edt"Email"=电子邮箱
 dlgQBugReportSend.tle=内部错误
 dlgQDockManagerClosedView.chk=不再显示
-dlgQDockManagerClosedView.fur=要再次重新打开它,请使用窗口菜单中的相应菜单项。
-dlgQDockManagerClosedView.hdl%1=You've closed the view '$1'.
-dlgQDockManagerClosedView.tle=Closed View
+dlgQDockManagerClosedView.fur=要重新打开它，请使用窗口菜单中相应的菜单项。
+dlgQDockManagerClosedView.hdl%1=你已经关闭了视图 $1。
+dlgQDockManagerClosedView.tle=关闭视图
 dlgQFileSaveAcceptFilterOverwrite.btn"Overwrite"=覆盖
-dlgQFileSaveAcceptFilterOverwrite.fur=要保存到其他文件，请单击“取消”。
-dlgQFileSaveAcceptFilterOverwrite.hdl%1=文件$1已存在。你确定覆盖它吗？
+dlgQFileSaveAcceptFilterOverwrite.fur=要保存到其他文件，请单击取消。
+dlgQFileSaveAcceptFilterOverwrite.hdl%1=文件 $1 已存在。你确定覆盖它吗？
 dlgQFileSaveAcceptFilterOverwrite.tle=覆盖文件
-dlgQIntegerInputProviderInvalidValue.fur%2=端口必须在$1到$2的范围内。
-dlgQIntegerInputProviderInvalidValue.hdl%1=字段“$1”中的文本无效。
+dlgQIntegerInputProviderInvalidValue.fur%2=端口必须在 $1 到 $2 的范围内。
+dlgQIntegerInputProviderInvalidValue.hdl%1=字段 $1 中的文本无效。
```

Comment:
The quotes here are also required.

I will not point out one by one in other places, please confirm and update.

Reply:
Thanks, I've fixed it.